Add a troubleshooting section for when the SDK appears not to be sending any data (#11818)
getsentry/sentry-docs: docs/platforms/javascript/common/troubleshooting/index.mdx

sidebar_order: 9000
---

<Expandable permalink title="The SDK is not sending any data">

> If you set up the Sentry SDK and it is not sending any data to Sentry,

Reviewer: ```suggestion
If you set up the Sentry SDK and it's not sending any data to Sentry:
```

- Check that you have configured a DSN and that you are passing it to the `dsn` option in `Sentry.init()`.

  If you are using environment variables to pass the DSN, make sure the environment variables are set in all relevant environments. Additionally, if you are using environment variables inside a framework, check that the framework will include the environment variables in your bundle. Often this means you will have to prefix your environment variables with special prefixes defined by your framework (`NEXT_PUBLIC_` in Next.js, `NUXT_` in Nuxt, `VITE_` for Vite projects, `REACT_APP_` for Create React App, ...).

- Check that you have disabled any ad-blockers.
- Set `debug: true` in the `Sentry.init()` options and observe your console output when you start your application. The SDK may tell you why it is not sending any data.

> - Check the "Stats" and "Subscription" pages in Sentry. You may have ran out of quota.

Reviewer: ```suggestion
- Check the [Stats](https://sentry.io/orgredirect/organizations/:orgslug/stats/) and [Suscription](https://sentry.io/orgredirect/organizations/:orgslug/settings/billing/overview/) pages in Sentry. You may have ran out of quota.
```
